.travis.yml

```diff
@@ -77,10 +84,13 @@ matrix:
            CONDA_DEPENDENCIES=${CONDA_ALL_DEPENDENCIES}
            EVENT_TYPE='cron'
 
+
 install:
   - git clone git://github.com/astropy/ci-helpers.git
-  - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
+  # - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
   # additional external tools (Issue #898) -- HOLE
+  - sudo pip install --upgrade-pip
+  - sudo pip install $PIP_DEPENDENCIES
```

Comment:
you can find in the logs that the sudo fails because you aren't allowed to write. The previous method was ok